content/docs/command-reference/cache/dir.md

```diff
@@ -24,22 +24,31 @@ Unlike doing so with `dvc config cache`, `dvc cache dir` transform paths
 **relative to the config file location**. However, if the `value` provided is an
 absolute path, then it's preserved as it is.
 
-If no path `value` is provided to this command, it prints the path for current
-cache directory.
+When reading (`value` is not provided) the value is read from the system,
+global, repo and local configuration files by default, and options `--system`,
+`--global`, `--repo`, and `--local` can be used to tell the command to read from
+only that location.
+
+When writing (`value` is provided or `--unset` is used), the new value is
+written to the projects configuration file by default (`.dvc/config`), and
+options `--system`, `--global` and `--local` can be used to tell the command to
+write to that location (you can say `--repo` but that is the default).
```

Comment:
I would move this up as part of the first paragraph or as a 2nd paragraph. Suggested text:
```suggestion
The `value` is written to the repo's config file by default (`.dvc/config`).
Options `--system`, `--global`, and `--local` can be used to write to the other
possible config files.
```

Reply:
> Options `--system`, `--global` ...

Same

Reply:
Again, adapted from git help, but ok let's tweak it.